### PR TITLE
Add search:query command line task

### PR DESCRIPTION
**Pull request by djjuhasz**
Add a command line `search:query` task to return a list of
Elasticsearch results for the given query.

### CODE REVIEW

File: lib/task/search/arSearchQueryTask.class.php [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+{
+    /**
+     * @see sfTask
+     *
+     * @param mixed $arguments
+     * @param mixed $options
+     */
+    public function execute($arguments = [], $options = [])
+    {
+        $count = 0;
```
Comment (mcantelon) [inline lib/task/search/arSearchQueryTask.class.php]: This seems unused.

File: lib/task/search/arSearchQueryTask.class.php [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+                $i + 1,
+                $resultSet->getTotalHits()
+            )
+        );
+    }
+
+    protected function outputSearchFields($className)
+    {
+        $count = 0;
+        $fields = $this->getFields($className);
```
Comment (mcantelon) [inline lib/task/search/arSearchQueryTask.class.php]: This function doesn't seem to exist (I get an error when I try to use this option).

**Review by mcantelon — COMMENTED** [review]
This'll be a handy tool! Found one issue, though.

File: lib/task/search/arSearchQueryTask.class.php (hunk 1 above)
Comment (djjuhasz) [reply lib/task/search/arSearchQueryTask.class.php]: Thanks, good catch. :)

File: lib/task/search/arSearchQueryTask.class.php (hunk 2 above)
Comment (djjuhasz) [reply lib/task/search/arSearchQueryTask.class.php]: Oops, yeah I updated the method name to `getSearchFields()` and forgot to update it here. 😊

Comment (mcantelon) [reply lib/task/search/arSearchQueryTask.class.php]: Ah, cool cool!

**Review by mcantelon — APPROVED** [review]
Looks good to me!

**Comment by djjuhasz** [comment]
This script is intended for development / analysis so it's probably better not to include it in the public release.

**Pull request closed without merging**